rules.services.yml

```diff
@@ -5,3 +5,8 @@ services:
   plugin.manager.rules_data_processor:
     class: Drupal\rules\Context\DataProcessorManager
     arguments: ['@container.namespaces', '@module_handler']
+  webprofiler.rules:
+    class: Drupal\rules\Webprofiler\DataCollector\RulesDataCollector
+    arguments: ['@logger.channel.rules']
+    tags:
+      - { name: data_collector, template:'@rules/Collector/rules.html.twig', id:'rules', title:'Rules', priority: 200}
```

Comment:
Can you add white spaces between template:, id:, title: and the respective values?